app/workers/alert_mailer_worker.rb

```diff
@@ -3,14 +3,14 @@
 class AlertMailerWorker
   include Sidekiq::Worker
 
-  def self.schedule_email(alert:, user:)
-    perform_async(alert.id, user.id)
+  def self.schedule_email(alert)
+    perform_async(alert.id, alert.target_user.id)
   end
 
   def perform(alert_id, user_id)
     alert = Alert.find(alert_id)
     user  = User.find(user_id)
-    
+
     AlertMailer.alert(alert, user).deliver_now
```

Comment:
We should just run alert.email_target_user inside the `perform`. The less it knows about it's job the better.